battleground.py

```diff
@@ -1,14 +1,14 @@
-
-from collections import defaultdict
+import copy
+import json
 import logging
 import threading
 import time
-import rules
-import json
-import copy
+from collections import defaultdict
 from datetime import datetime
 from pathlib import Path
-from enumeration import RoundWinner, RoundAdvantage
+
+import rules
+from enumeration import RoundWinner, RoundAdvantage, GamesListFileField, BotMessageField
```

Comment:
Class names are too long or need to be shadowed.
Maybe use:
```
from enumeration import RoundWinner, RoundAdvantage, \
GamesListFileField as gFields, BotMessageField as bmFields
```
Or some better nicknames for these classes.